misc/config_tools/data/generic_board/generic_code/hybrid/pci_dev.c

```diff
@@ -4,11 +4,11 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-#include <vm_config.h>
+#include <x86/vm_config.h>
```

Comment:
@NanlinXie  @wenlingz  Do we hold this PR and wait for 5976 ? It requires header include path change. This conflict would cause a build failure in between if someone bisect.